test/testutil/stanza.c

```diff
@@ -88,30 +88,31 @@ int test_readstanza(STANZA *s)
     PAIR *pp = s->pairs;
     char *p, *equals, *key;
     const char *value;
+    static char buff[131072];
 
-    for (s->numpairs = 0; BIO_gets(s->fp, s->buff, sizeof(s->buff)); ) {
```

Comment:
Once rebasing onto master is fixed for this branch.. Maybe this should reduce the structures buff[]

Reply:
??? Not sure what you mean here.

Reply:
The buffer that you are no longer using should be reduced or removed from the struct. We made it large to account for PQ.

Reply:
I'm sure I did this already but it must have been lost somewhere.  Removed now.